test/forge/integration/AuthorizationIntegrationTest.sol

```diff
@@ -127,14 +127,23 @@ contract AuthorizationIntegrationTest is BaseTest {
         authorization.nonce = 0;
         authorization.authorizer = vm.addr(privateKey);
 
+        address authorized = _boundAddressNotZero(authorization.authorized);
+        authorization.authorized = address(0);
```

Comment:
We should create an independent test I think

Reply:
But it would essentially be the same test whereas we can test both at the same time

Reply:
Then can we move `address authorized = _boundAddressNotZero(authorization.authorized);` lower. Else it seems weird to query it now and set `authorization.authorized` to the address zero.

Reply:
i pushed a split because you were right it wasn't testing the same thing